Frontend/src/app/shared/indicator-verdict-hero/indicator-verdict-hero.component.html

```
@@ -0,0 +1,145 @@
+<section class="hero" [class]="tierCssClass(verdict().tier)">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)